src/ArmTemplates/Common/API/Clients/Abstractions/ApiClientBase.cs

```diff
@@ -30,11 +31,18 @@ public ApiClientBase(IHttpClientFactory httpClientFactory)
             this.httpClient = httpClientFactory.CreateClient();
         }
 
-        protected async Task<string> CallApiManagementAsync(string azToken, string requestUrl, bool useCache = true, ClientHttpMethod method = ClientHttpMethod.GET)
+        /// <summary>
+        ///  Main method to call trigger api management emdpoint.
```

Comment:
typo "endpoint". nit: I dont think that is a comment, that really tells something useful. It is pretty understandable that this is an API to call apim

Reply:
will update the endpoint type. 
Regarding another nit: Do you want me to delete the description? 
Mainly it was introduced for listing output of the method